org.winehq.Wine.yml

```diff
@@ -22,9 +22,12 @@ finish-args:
   - --filesystem=xdg-music
   - --filesystem=xdg-videos
   - --filesystem=xdg-download
+  - --filesystem=~/Games:create
+  - --filesystem=~/GOG Games:create
```

Comment:
A left-over? This app should have only permissions essential for wine itself to function properly.

Reply:
Basically nothing will work until user explicitly will add this dirs. We can't predict all paths but this most common locations. Whole app about running games and there is no any dirs where games stored allowed by default in current setup which looks very odd.

Reply:
Well, this app is about being a feature-full Wine flatpak build, probably reusable as a baseapp for other apps. It's not intended to be a game launcher for end users. Apps basing on it should add filesystem permissions instead.

Reply:
Reasonably. I though until we will have at least one app or launcher this will make sense.